.github/workflows/tests.yml

```diff
@@ -73,3 +73,9 @@ jobs:
         with:
           file: ./coverage.txt
         if: env.GIT_DIFF
+  
+  test_persistence:
```

Comment:
@tychoish is this the right place for this to live?

Reply:
oh I have no idea. I think we have more of these "run a shell script integration tests" in the "build.yml" file so I'd be inclined to do it there. 

I don't think this task will necessarily have the checkout/artifacts built as is written now.